test/regression.js

```diff
@@ -112,14 +141,15 @@ const runTests = async (list) => {
       }
 
       if (req.url.startsWith('/original/')) {
+        stats[statsName].lengthOrig = file.length;
         res.setHeader('Content-Type', 'image/svg+xml');
         res.end(file);
         return;
       }
       if (req.url.startsWith('/optimized/')) {
-        const optimized = optimize(file, {
-          floatPrecision: 4,
-        });
+        const optimized = optimize(file, CONFIG);
+        stats[statsName].lengthOpt = optimized.data.length;
```

Comment:
Is this actually useful? I shouldn't store stats for the sake of storing stats.

I'd imagine it's only helpful to store the overall bytes reduced, or the average percentage reduced across files. There's no need to store the difference between all files, though.

In the case that someone is focused on a specific file, they can pass the argument to limit files tested.